### PR TITLE
[PROJQUAY-795] Monkey patch top of remaining gunicorn entrypoints

### PR DESCRIPTION
**Pull request by kurtismullins**
### Description of Changes

Full coverage of monkey patching for gunicorn.

#### Changes:

Includes monkey patching at the top of remaining gunicorn entrypoint files. This is consistent with the other `gunicorn*.py` files.

#### Issue:

- https://issues.redhat.com/browse/PROJQUAY-795

**TESTING**

- Run the container. Ensure nothing related to verbs is broken.

**BREAKING CHANGE**

n/a

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: conf/gunicorn_verbs.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# NOTE: Must be before we import or call anything that may be synchronous.
+from gevent import monkey
```
Comment (josephschorr) [inline conf/gunicorn_verbs.py:2]: We don't use gevent in verbs, so this isn't needed

Comment (kurtismullins) [reply conf/gunicorn_verbs.py:2]: @josephschorr In that case, would you be opposed to me removing the `gunicorn_verbs.py` file and pointing supervisord to start verbs as a standard python module?

Current:
```
command=nice -n 10 gunicorn -c %(ENV_QUAYCONF)s/gunicorn_verbs.py verbs:application
```

Proposed (untested)
```
python -m verbs:application
```

Comment (josephschorr) [reply conf/gunicorn_verbs.py:2]: No. It still uses *gunicorn*, just not *gevent*. It uses sync workers.

Comment (kurtismullins) [reply conf/gunicorn_verbs.py:2]: Thanks! Closing this accordingly.

**Comment by kurtismullins** [comment]
The verbs module uses the synchronous worker and therefore does not require monkey patching. Closing this Pull Request accordingly.

**Pull request closed without merging**